charts/scalardb-mcp-server/templates/scalardb-mcp-server/service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "scalardb-mcp-server.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "scalardb-mcp-server.labels" . | nindent 4 }}
+  {{- with .Values.scalardbMcpServer.service.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.scalardbMcpServer.service.type }}
+  selector:
+    {{- include "scalardb-mcp-server.selectorLabels" . | nindent 4 }}
+  ports:
+    - name: http
+      port: {{ .Values.scalardbMcpServer.service.port }}
+      targetPort: {{ .Values.scalardbMcpServer.service.targetPort }}
+      protocol: {{ .Values.scalardbMcpServer.service.protocol }}
```

Comment:
It seems that the concrete method to configure service port settings is a bit different from other charts. For example, the Scalar Manager Helm Chart has a bit of a differencemethod:

- https://github.com/scalar-labs/helm-charts/blob/scalar-manager-3.0.0/charts/scalar-manager/templates/scalar-manager/service.yaml

To make Scalar Helm Charts consistent as long as possible, I think it would be better to use the same way as other charts to configure service post configurations.

Of course, if you have some special reason why you want to use this style configuration on the ScalarDB MCP Server helm chart side, I think we can use the current style. Do you have any concrete reasons?